torch/ao/quantization/fx/_decomposed.py

```diff
@@ -200,16 +201,13 @@ def choose_qparams_tensor(
        zero_point (int): quantization parameter for the target quantized Tensor
     """
     assert input.dtype == torch.float32, f"Expecting input to have dtype torch.float32, but got dtype: {input.dtype}"
-    assert quant_min < quant_max, f"Expecting quant_min to be smaller than quant_max but received min: {quant_min} max: {quant_max}"
+    assert qmin < qmax, f"Expecting quant_min to be smaller than quant_max but received min: {qmin} max: {qmax}"
```

Comment:
maybe we can reuse the validation code as well: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L272